Python/activation.py

```diff
@@ -25,4 +36,35 @@ def forward(self,x):
         normalised_score = exp_values / np.sum(exp_values,keepdims=True,axis=1)
         self.output = normalised_score
         return self.output 
+    
+class Sigmoid(Activation):
+    """
+    Sigmoid function is an activation function used in neural networks to introduce non-linearity into the model.
+    Output in the range (0, 1)
+    """
+
+    def forward(self, x):
+        self.output = 1 / (1 + np.exp(-x))
+        return self.output
+    
+class Tanh(Activation):
+    """
+    Hyperbolic tangent function is an activation function used in neural networks to introduce non-linearity into the model.
+    Output in the range (-1, 1)
+    """
+    def forward(self, x):
+        self.output = np.tanh(x)
+        return self.output
+    
+class ELU(Activation):
+    """
+    Exponential Linear Unit (ELU) is an activation function used in neural networks to introduce non-linearity into the model.
+    Output in the range (0, infinity)

```

Comment:
this output range (0,∞) is not accurate 
```suggestion
    Output in the range (0, infinity)

```
 the overall output range is (-alpha, infinity)